src/handlers/admin_v1_configs_delete.cpp

```diff
@@ -17,9 +19,11 @@ struct RequestData {
 };
 
 RequestData ParseRequest(const userver::formats::json::Value &request) {
```

Comment:
Let's delete `RequestData` and use the code-generated structs directly

Reply:
Ok, done